### ISSUE TITLE
Timer finish doesn't update accurately

### ISSUE THREAD

**Issue by petevb** [NewOrbit/onetime-cli#7]
From Skype / @Jameskmonger :

> Found an issue with onetime. Numbers used below are arbitrary as an example

> Start a Harvest timer linked to a TP task. Wait for the timer to hit 1 minute and type "onetime t finish". When onetime asks you which timer to finish, wait on that page for 1 minute. Scroll down to the timer (should say 1 minute but on Harvest it will now be 2 minutes) and finish it.

> 2 minutes will be logged on Harvest while 1 minute will be logged on TP.

**Comment by petevb** [comment]
Suggest won't fix.

We (@flytzen / @philoUK / Tom Dean ?) need to agree process for Issues/Features.  I see we've now got a project in TP too; we want to understand if there are aspirations to develop `onetime` further.

**Comment by petevb** [comment]
Oops - Before suggesting that we don't fix it I should've understood whether it's a real issue for James or others. It seems like an edge case to me, but maybe I don't use it the same way as others do.

**Comment by Jameskmonger** [comment]
@petevb I discovered this like so:

1. Go to end my timer "A" after 30 minutes
2. Realise I need to do 10 minutes more work, leave onetime on the "choose a timer" screen
3. Finish the work and finish the timer

30 mins logged to TP, 40 mins logged to Harvest

**Comment by petevb** [comment]
Yeah, don't do that :)  

Do you think it's something that probably won't happen again now you know about the bug, iykwim?

**Comment by Jameskmonger** [comment]
Probably won't happen again but if it's not much work it may be good to fix?

**Comment by flytzen** [comment]
Should fix, but leave it for someone to be annoyed enough to do it ;)
